content/docs/introducing-jsx.md

```diff
@@ -87,37 +88,37 @@ function getGreeting(user) {
 }
 ```
 
-### Specifying Attributes with JSX {#specifying-attributes-with-jsx}
+### Especificando Atributos com JSX {#specifying-attributes-with-jsx}
 
-You may use quotes to specify string literals as attributes:
+Você pode usar aspas para especificar strings literais como atributos:
 
 ```js
 const element = <div tabIndex="0"></div>;
 ```
 
-You may also use curly braces to embed a JavaScript expression in an attribute:
+Você também pode usar chaves para incorporar uma expressão JavaScript em um atributo:
 
 ```js
 const element = <img src={user.avatarUrl}></img>;
 ```
 
-Don't put quotes around curly braces when embedding a JavaScript expression in an attribute. You should either use quotes (for string values) or curly braces (for expressions), but not both in the same attribute.
+Não envolva chaves com aspas quando estiver incorporando uma expressão JavaScript em um atributo. Você deveria ou usar aspas (para valores em string) ou chaves (para expressões), mas não ambos no mesmo atributo.
 
->**Warning:**
+>**Atenção:**
 >
->Since JSX is closer to JavaScript than to HTML, React DOM uses `camelCase` property naming convention instead of HTML attribute names.
+>Como JSX é mais próximo de JavaScript que do HTML, o React DOM usa `camelCase` como convenção para nomes de propriedades ao invés dos nomes de atributos do HTML.
 >
->For example, `class` becomes [`className`](https://developer.mozilla.org/en-US/docs/Web/API/Element/className) in JSX, and `tabindex` becomes [`tabIndex`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/tabIndex).
+>Por exemplo, `class` se transforma em [`className`](https://developer.mozilla.org/pt-BR/docs/Web/API/Element/className) em JSX, e `tabindex` se transforma em [`tabIndex`](https://developer.mozilla.org/pt-BR/docs/Web/API/HTMLElement/tabIndex).
 
-### Specifying Children with JSX {#specifying-children-with-jsx}
+### Especificando Filhos com JSX {#specifying-children-with-jsx}
```

Comment:
I believe we will need to discuss in the glossary how to translate _"children"_. Like _"state"_ and _"props"_, it's a main concept of React. Maybe something like _"elementos filhos"_. Just _"Filhos"_ seems odd to me.

Reply:
"_children_" translated as "_filhos_" seems odd to me also, but it is the term used in Computer Science for [data structure](https://pt.wikipedia.org/wiki/%C3%81rvore_(estrutura_de_dados)).

Change it to "_elementos filhos_" seems more accurate to its meaning.